server/server/src/upload_fridge_tag.rs

```diff
@@ -1,34 +1,30 @@
-use std::path::Path;
+use std::ops::Deref;
 
-use actix_multipart::form::{tempfile::TempFile, MultipartForm};
+use actix_multipart::form::MultipartForm;
 use actix_web::{
     post,
     web::{self, Data},
-    HttpResponse,
+    HttpRequest, HttpResponse,
 };
 use anyhow::Context;
 
 use serde::Deserialize;
 
 use service::{
+    auth_data::AuthData,
     sensor::berlinger::{read_sensor, ReadSensor},
     service_provider::ServiceProvider,
     settings::Settings,
+    static_files::{StaticFileCategory, StaticFileService},
 };
-use util::prepare_file_dir;
+use util::format_error;
 
-const TEMP_FRIDGETAG_FILE_DIR: &str = "fridge_tag";
```

Comment:
don't need special fridge_tag folder, using static file service and temporary file, which will be removed